### PR TITLE
Add a 'Pull requests' link and build hive status integration to plugin summary

### PR DESCRIPTION
**Pull request by lacostej**

### CODE REVIEW

**Comment by kohsuke** [comment]
I couldn't merge POM changes cleanly, so I hand picked your changes in JenkinsPluginInfoMacro and merged them.

**Comment by lacostej** [comment]
Thanks for picking that one up :)

**Comment by orrc** [comment]
The problem with the badge is that it shows the last build result, which may have been triggered by a pull request.

e.g. Looking at the most recent failure on the plugin build server, it makes it look like the Python plugin doesn't build:
https://wiki.jenkins-ci.org/display/JENKINS/Python+Plugin

File: src/main/java/org/jenkinsci/confluence/plugins/JenkinsPluginInfoMacro.java [hunk 1]
```diff
@@ -157,6 +157,10 @@ public String execute(Map parameters, String body, RenderContext renderContext)
                                         .append("]\n[Since Latest Release|").append(fisheyeBaseUrl)
                                         .append(releaseTimestamp).append(fisheyeEndUrl).append(']');
                         }
+                        if (isGithub) {
+                            String buildhiveUrl = "https://buildhive.cloudbees.com/job/jenkinsci/job/" + sourceDir;
+                            toBeRendered.append("[!").append(buildhiveUrl).append("/badge/icon|border=0!|").append(buildhiveUrl).append("/]");
```
Comment (KostyaSha) [inline src/main/java/org/jenkinsci/confluence/plugins/JenkinsPluginInfoMacro.java]: Please remove this badge icon from macros, any failed PR shows failed icon and i see no way to disable it.

**Comment by KostyaSha** [comment]
:-1:

**Comment by kohsuke** [comment]
https://issues.jenkins-ci.org/browse/JENKINS-25008 would let us add this back.

**Pull request closed without merging**